src/main/resources/data/purchase-orders/S60402_ongoing_closed.json

```diff
@@ -15,5 +15,8 @@
     "reviewPeriod": 30
   },
   "vendor": "80fb5168-cdf1-11e8-a8d5-f2801f1b9fd1",
-  "workflowStatus": "Closed"
+  "workflowStatus": "Closed",
+  "closeReason": {
+    "reason": "Wrong order"
```

Comment:
I think the reason for closing should be "Complete" as there is a [paid invoice](https://github.com/folio-org/mod-invoice-storage/blob/master/src/main/resources/data/invoices/30121_paid.json) for this order.